components/EditRecipeComponents/Title.js

```diff
@@ -68,6 +68,7 @@ const Title = ({ title, currentActive }) => {
                         }}
                         style={styles.title}
                         multiline
+                        maxLength={23}
```

Comment:
In createRecipe the max length is 55, I would consider changing the max here to be the same.